catalog/admin/module/catalog-admin-module-sources/src/main/webapp/templates/sourceModal.handlebars

```diff
@@ -11,34 +11,72 @@
  *
  **/
  --}}
- 
+
 <div class="modal-dialog">
     <div class="modal-content">
         <div class="modal-header">
             <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
             <h4 class="modal-title">
                 {{#is mode 'edit'}}
-                    Edit {{name}}
+                    Edit:  {{name}}
                 {{else}}
                     Add Source
                 {{/is}}
             </h4>
         </div>
+        <div class="modal-organization container-fluid">
+        </div>
         <div class="modal-body">
             <form class="form-horizontal add-federated-source container-fluid">
                 <fieldset>
                     <div class="sourceName"></div>
                     <div class="bound-controls">
                         <div class="control-group">
-                            <label class="control-label">Source Type</label>
-                            <select class="form-control input-sm sourceTypesSelect"></select>
+
+                            <select class="form-control input-sm activeBindingSelect"></select>
+
                         </div>
                         <div class="modal-details"></div>
                     </div>
                 </fieldset>
             </form>
-        </div>
 
+            <label class="control-label2 container-fluid">Binding Configurations:</label>
+            <div class="modal-accordions container-fluid">
+            </div>
+            <label class="control-label2  container-fluid">Reports:</label>
+            <div class="modal-reports container-fluid">
+                {{#if reportActions}}
+                    <ul class="report-list">
+                        {{#each reportActions}}
+                            <li>
+                                <a href="{{url}}">{{title}}</a>
```

Comment:
`<a href="{{url}}" target="_blank">{{title}}</a>`
Should make report open in new tab